### ISSUE TITLE
train custom dataset， low score

### ISSUE THREAD

**Issue by betterhalfwzm** [zylo117/Yet-Another-EfficientDet-Pytorch#79]
After 5 epochs, i test images, the score is low, only  set very low threshold =0.1 can detect it, but test mAP looks normal.  Same image, less epochs, official EfficientDet score is high.
Can you give me some advice? Thx!!!!!!!!!

**Comment by zylo117** [comment]
1. it is the latest code?
2. did you run and succeed getting a good result on the tutorial?
3. can you kmeans your anchors ratios?
4. pls share your loss graph, logs, hyperparameters

**Comment by betterhalfwzm** [comment]
> 1. it is the latest code?
> 2. did you run and succeed getting a good result on the tutorial?
> 3. can you kmeans your anchors ratios?
> 4. pls share your loss graph, logs, hyperparameters

1. yes
2. yes
3. no, the same with you and official EfficientDet 
4. 
python train.py -c 0 --batch_size 64 --lr 1e-5 --num_epochs 10 \
 --load_weights weights/efficientdet-d0.pth

![image](https://user-images.githubusercontent.com/20449990/79292770-82d7d100-7f04-11ea-8e2b-7cf94b447d82.png)
![image](https://user-images.githubusercontent.com/20449990/79292776-88351b80-7f04-11ea-955f-9037a84bf587.png)
![image](https://user-images.githubusercontent.com/20449990/79292784-8e2afc80-7f04-11ea-8e68-3e1c444133db.png)
@zylo117

**Comment by zylo117** [comment]
try this one:
`python train.py -c 0 --batch_size 64 --lr 1e-5 --num_epochs 10  --head_only True --debug True`

then check out test/ folder, see if there any positive boxes on the images

**Comment by betterhalfwzm** [comment]
> try this one:
> `python train.py -c 0 --batch_size 64 --lr 1e-5 --num_epochs 10 --head_only True --debug True`
> 
> then check out test/ folder, see if there any positive boxes on the images

boxes is right, but score is low

**Comment by zylo117** [comment]
> > try this one:
> > `python train.py -c 0 --batch_size 64 --lr 1e-5 --num_epochs 10 --head_only True --debug True`
> > then check out test/ folder, see if there any positive boxes on the images
> 
> boxes is right, but score is low

boxes with the correct label?
Then keep training.

**Comment by ancorasir** [comment]
@betterhalfwzm ， my experience is you might want to increase the learning rate to 10^(-4). The loss will decrease much more quickly. I have tested it for my custom dataset training and it works well.

**Comment by fightingaaa** [comment]
Have you solved this problem? I met it , the MAP is high when thred=0.1,but score is very low
